README.md

```diff
@@ -390,6 +391,17 @@ test('passes when any value in array pass given predicate', () => {
 });
 ```
 
+#### .toBeWithinRange(min, max)
+
+Use `.toBeWithinRange` when you want to check if array contains elements within a given range.
```

Comment:
We should probably document here that the numbers are both inclusive.

Reply:
@keeganwitt I have updated it.
Thanks!